src/notebookConverter.ts

```diff
@@ -37,8 +37,12 @@ export class NotebookConverterImpl implements IDisposable {
             }
         }
 
-        if (uri.scheme === InteractiveScheme) {
-            return uri.path.toLowerCase();
+        if (uri.path.endsWith('.interactive')) {
```

Comment:
seems like we could still just `return uri.path.toLowerCase();`?

Reply:
Paths start with `/`, these values don't, I assumed to make them distinguishable from real paths (the case on line 52)